src/main/kotlin/com/jetbrains/snakecharm/codeInsight/SnakemakeAPI.kt

```diff
@@ -152,7 +155,8 @@ object SnakemakeAPI {
             SECTION_WRAPPER,
             SECTION_VERSION, SECTION_THREADS,
             SECTION_PRIORITY, SECTION_SINGULARITY, SECTION_CACHE,
-            SECTION_CONTAINER, SECTION_NOTEBOOK, SECTION_ENVMODULES
+            SECTION_CONTAINER, SECTION_CONTAINERIZED, SECTION_NOTEBOOK,
```

Comment:
Have you checked it on runtime? Is it true, or just an assumption?

Reply:
judging by the documentation, containerized has absolutely the same behavior as the container

Reply:
Thx, I've also checked the runtime behaviour and agree with you.